BlogTemplate/Pages/Edit.cshtml

```diff
@@ -1,6 +1,6 @@
 @page "{slug}"
-@using BlogTemplate
-@using BlogTemplate.Models

```

Comment:
I fixed this in my recent commit, but for some reason this comment isn't going away. I addressed it though